docs/Contribute/ContributeTranslate.md

```diff
@@ -1,10 +1,13 @@
-# Translate BTCPay Server
+# Why translations matter
```

Comment:
Can we but Translating BTCPay Server as h1 here (#title)? H1 is usually the main header and would help us with SEO. Rest I'd put as H2 and for h2's we already have adjust accordingly to H3 (###title)

Reply:
So basically re-arange translating and why it matters while keeping first h1.

Reply:
Resolved by 58d915f